tests/boot/cloud_init.pm

```diff
@@ -10,9 +10,11 @@ use Mojo::Base 'opensusebasetest';
 use testapi;
 use serial_terminal 'select_serial_terminal';
 
-sub upload_journal {
+sub post_boot_logs {
     script_run("journalctl --no-pager -o short-precise > /tmp/journal_after_boot.log");
-    upload_logs "/tmp/journal_after_boot.log";
+    upload_logs("/tmp/journal_after_boot.log");
```

Comment:
You can merge these lines using lib/Utils/Logging.pm::save_and_upload_log()

Reply:
done